### PR TITLE
Remove the script from db if it wasn't found

### PR DESCRIPTION
**Pull request by Waqar144**

### CODE REVIEW

File: src/services/scriptingservice.cpp [hunk 1]
```diff
@@ -288,12 +290,11 @@ void ScriptingService::initComponents() {
     clearCustomStyleSheets();
     _scriptComponents.clear();
     _settingsVariables.clear();
-    QList<Script> scripts = Script::fetchAll();
+    bool enabledOnly = true;
```
Comment (pbek) [inline src/services/scriptingservice.cpp:293]: Do we need `enabledOnly`?

Comment (Waqar144) [reply src/services/scriptingservice.cpp:293]: Not really, but we check for it in the loop anyways. So I thought to just fetch only the enabled scripts in the first place and remove the check inside loop

Comment (pbek) [reply src/services/scriptingservice.cpp:293]: No, I meant "do we need the variable"? 😁

Comment (Waqar144) [reply src/services/scriptingservice.cpp:293]: Hahah. No, but the reads like a story with it 🤣

Comment (pbek) [reply src/services/scriptingservice.cpp:293]: Clion shows me the names of method parameters. 😆 
And YOU, eager to get the last bit of a byte optimized, introduce a variable that is only used once and never changes. 😆

Comment (Waqar144) [reply src/services/scriptingservice.cpp:293]: It will be optimized by the compiler i am sure :wink: :laughing:

Comment (pbek) [reply src/services/scriptingservice.cpp:293]: Seeing that variable makes my head spin. Can't you just write a comment over the function call? 😁

Comment (Waqar144) [reply src/services/scriptingservice.cpp]: Okay done.

File: src/services/scriptingservice.cpp [hunk 2]
```diff
@@ -288,12 +290,12 @@ void ScriptingService::initComponents() {
     clearCustomStyleSheets();
     _scriptComponents.clear();
     _settingsVariables.clear();
-    QList<Script> scripts = Script::fetchAll();
 
-    Q_FOREACH (Script script, scripts) {
-        if (script.isEnabled()) {
-            initComponent(script);
-        }
+    //fetch enabled only
```
Comment (pbek) [inline src/services/scriptingservice.cpp:294]: ...with a space space after `//`, like everywhere else, pleeeease 😁

Comment (Waqar144) [reply src/services/scriptingservice.cpp]: done

Comment (pbek) [reply src/services/scriptingservice.cpp]: Thank you 😁

**Pull request merged**